### PR TITLE
colours from style guide

### LINKED ISSUE #1116: Add app colours to theme
https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b76dd216617eb79202cacd2

image

### PR DESCRIPTION
**Pull request by sastels**
resolves #1116

### CODE REVIEW

File: theme.js [hunk 1]
```diff
@@ -16,6 +16,18 @@ const theme = {
     sm: "960px",
     md: "1280px",
     lg: "1920px"
+  },
+  color: {
```
Comment (smcmurtry) [inline theme.js]: what country do you think we live in?!

Comment (sastels) [reply theme.js]: @maxneuvians we need your sober (or not) judgement here.

Comment (maxneuvians) [reply theme.js]: https://github.com/HashanP/postcss-spiffing

Comment (maxneuvians) [reply theme.js]: but the colour names are spot on

**Comment by pcraig3** [comment]
Yes!

https://github.com/cds-snc/ircc-rescheduler/blob/master/web/src/styles.js#L44

**Comment by sastels** [comment]
lol how did @pcraig3 find out about this?

**Review by smcmurtry — APPROVED** [review]
👍

**Pull request merged**